.github/workflows/deploy_publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Publish
+on:
+  push:
+    branches: [master]
+    paths-ignore:
+      - "**.md"
+  repository_dispatch:
+    types: ["default-config-trigger"]
+
+env:
+  APPOINTMENTS_VERSION: v2
```

Comment:
Can we use semantic versioning here ? Because the docker image name already says it is v2. Also we could make it consistent by putting the version in .appversion file.  Refer [this](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/.github/workflows/build_publish.yml#L17) for an example.